src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1382,6 +1385,9 @@ impl AstDisplay for DropClusterReplicasStatement {
             f.write_str("IF EXISTS ");
         }
         f.write_node(&display::comma_separated(&self.names));
+        if self.cascade {
```

Comment:
I don't know how to actually exercise this code, but it seems trivially correct.

Reply:
You can add a test to `src/sql-parser/tests/testdata/ddl`

Reply:
added!